### PR TITLE
 etcd: move etcd service to openshift-etcd

### PR DESCRIPTION
**Pull request by deads2k**
https://github.com/openshift/machine-config-operator/pull/648 has merged.  This updates the installer to move resources to the openshift-etcd namespace.  The consumers have already been updated and resources exactly match the ones that they're making to avoid thrashing.

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
This is only moving the service. please reflect the commit message and pr title to match that.

**Comment by deads2k** [comment]
> This is only moving the service. please reflect the commit message and pr title to match that.

updated

File: pkg/asset/templates/content/bootkube/host-etcd-service.go [hunk 1]
```diff
@@ -12,25 +12,25 @@ const (
...
 // Dependencies returns all of the dependencies directly needed by the asset
-func (t *HostEtcdServiceKubeSystem) Dependencies() []asset.Asset {
+func (t *HostEtcdServiceOpenshiftEtcd) Dependencies() []asset.Asset {
 	return []asset.Asset{}
 }
 
 // Name returns the human-friendly name of the asset.
-func (t *HostEtcdServiceKubeSystem) Name() string {
+func (t *HostEtcdServiceOpenshiftEtcd) Name() string {
 	return "HostEtcdServiceKubeSystem"
```
Comment (wking) [inline pkg/asset/templates/content/bootkube/host-etcd-service.go]: This needs to get bumped too.

File: pkg/asset/templates/content/bootkube/etcd-service.go [hunk 2]
```diff
@@ -12,25 +12,25 @@ const (
...
 // Name returns the human-friendly name of the asset.
-func (t *EtcdServiceKubeSystem) Name() string {
+func (t *EtcdServiceOpenshiftEtcd) Name() string {
 	return "EtcdServiceKubeSystem"
 }
 
 // Generate generates the actual files by this asset
-func (t *EtcdServiceKubeSystem) Generate(parents asset.Parents) error {
+func (t *EtcdServiceOpenshiftEtcd) Generate(parents asset.Parents) error {
 	fileName := etcdServiceKubeSystemFileName
```
Comment (wking) [inline pkg/asset/templates/content/bootkube/etcd-service.go]: This needs to get bumped.

File: pkg/asset/templates/content/bootkube/etcd-service.go [hunk 3]
```diff
@@ -12,25 +12,25 @@ const (
...
 // Dependencies returns all of the dependencies directly needed by the asset
-func (t *EtcdServiceKubeSystem) Dependencies() []asset.Asset {
+func (t *EtcdServiceOpenshiftEtcd) Dependencies() []asset.Asset {
 	return []asset.Asset{}
 }
 
 // Name returns the human-friendly name of the asset.
-func (t *EtcdServiceKubeSystem) Name() string {
+func (t *EtcdServiceOpenshiftEtcd) Name() string {
 	return "EtcdServiceKubeSystem"
```
Comment (wking) [inline pkg/asset/templates/content/bootkube/etcd-service.go]: This needs to get bumped.

**Comment by wking** [comment]
[e2e-aws][1]:

```
level=fatal msg="waiting for Kubernetes API: context deadline exceeded"
```

Hrm.

/retest

[1]: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_installer/1665/pull-ci-openshift-installer-master-e2e-aws/5527

**Pull request merged**